ci/ray_ci/container.py

```diff
@@ -99,3 +106,11 @@ def get_run_command_extra_args(
         gpu_ids: Optional[List[int]] = None,
     ) -> List[str]:
         pass
+
+    @abc.abstractmethod
+    def get_artifact_mount_host(self) -> str:
+        pass
+
+    @abc.abstractmethod
+    def get_artifact_mount_container(self) -> str:
+        pass
```

Comment:
make it one method that returns a tuple?

Reply:
there are some use cases later on that I need one and not the other

Reply:
in those cases you can just drop the other return value?

the two values are so

Reply:
sound good, let's do that